.github/workflows/ir-llm-benchmark.yml

```diff
@@ -0,0 +1,85 @@
+name: SageMaker PythonSDK Integration Tests
+
+on:
+   schedule:
+     - cron: '0 17 * * 5' # Run every Friday at 5pm
+
+jobs:
+  create-runners:
+    runs-on: [self-hosted, scheduler]
+    steps:
+      - name: Create new CPU instance
+        id: create_cpu1
+        run: |
+          cd /home/ubuntu/djl_benchmark_script/scripts
+          token=$( curl -X POST -H "Authorization: token ${{ secrets.ACTION_RUNNER_PERSONAL_TOKEN }}" \
+          https://api.github.com/repos/deepjavalibrary/djl-serving/actions/runners/registration-token \
+          --fail \
+          | jq '.token' | tr -d '"' )
+          ./start_instance.sh action_cpu $token djl-serving
+    outputs:
+      cpu_instance_id1: ${{ steps.create_cpu1.outputs.action_cpu_instance_id }}
+      
+
+  endpoint-tests:
+    runs-on: [ self-hosted, cpu ]
+    timeout-minutes: 120
+    needs: create-runners
+    strategy:
+      fail-fast: false
+      matrix:
+        engine: [lmi-dist, trtllm]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python3
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.10.x'
+      - name: Install pip dependencies
+        run: pip3 install -U boto3 awscli
+      - name: Install IR-LLM
+        working-directory: tests/integration/benchmark/ir-llm/sagemaker
+        run: |
+          pip3 install --quiet sagemaker-2.227.1.dev0-py3-none-any.whl
+          aws configure add-model --service-model file://sagemaker-2017-07-24.normal.json  --service-name sagemaker
```

Comment:
Can we put this wheel and service model in s3? At least for the wheel, we shouldn't have that as part of version control in my opinion.